sdk/batch/azure-compute-batch/src/test/java/com/azure/compute/batch/JobScheduleTests.java

```diff
@@ -117,15 +120,24 @@ public void canCRUDJobSchedule() throws Exception {
             Assertions.assertEquals("key1", jobSchedule.getMetadata().get(0).getName());
             Assertions.assertEquals((Integer) 100, jobSchedule.getJobSpecification().getPriority());
 
-            // DELETE
-            batchClient.deleteJobSchedule(jobScheduleId);
+            // DELETE using LRO
+            SyncPoller<BatchJobSchedule, Void> poller = batchClient.beginDeleteJobSchedule(jobScheduleId);
```

Comment:
Poller in test would be wrapped in a `setPlaybackSyncPollerPollInterval`, so that playback takes less them.
https://github.com/Azure/azure-sdk-for-java/wiki/TypeSpec-Java-Quickstart#recordplayback-tests